ofl/ysabeauinfant/DESCRIPTION.en_us.html

```diff
@@ -13,5 +13,5 @@
   <li><i>Ysabeau SC</i> replaces the lowercase with small caps.</li>
 </ul> 
 <p>
-    To contribute, please see <a href="https://github.com/CatharsisFonts/Ysabeau">github.com/CatharsisFonts/Ysabeau</a>.
-</p>
+    To contribute, please see <a href="github.com/CatharsisFonts/Ysabeau">https://github.com/CatharsisFonts/Ysabeau</a>.
```

Comment:
I think this is something I already fixed when the previous PRs were made; it should be either refected in the file upstream, or the description removed for upstream.yaml to not have it again

Reply:
I'll fix it upstream or else it will return again.